### PR TITLE
Update What's New Page 10/1

### LINKED ISSUE #1894: Update What's New Page 10/1
A place to collect updates that will be published to the Design System What's New page every other week. Comment on this issue with updates you think should be published on the What's New page.

Once the page has been updated at the end of 2 weeks, this issue will be closed and a new issue opened for the next 2 weeks.

### PR DESCRIPTION
**Pull request by jcfilben**
#### What does this PR do?
Updates What's New page

#### Where should the reviewer start?
whats-new.mdx 

#### What are the relevant issues?
Closes #1894

#### Should this PR be mentioned in Design System updates?
No

#### Is this change backwards compatible or is it a breaking change?
Backwards compatible

### CODE REVIEW

File: aries-site/src/pages/whats-new.mdx [hunk 1]
```diff
@@ -52,6 +52,10 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## September 20 - October 1, 2021
+- New resources and a checklist has been added to the [Accessibility page](/foundation/accessibility).
+- Fixed height on [Table](/components/table) examples when in full screen.
+
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:58]:
```suggestion
- Completed [Button size designs](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=0%3A1).
- Completed ideation and collected feedback for [NameValueList designs](https://www.figma.com/file/Ny7rYK5gI5VO4PtZsq6vGC/HPE-Name-Value-Pair?node-id=148%3A1936).
- Provides "request for comments" pull requests for Grommet NameValueList component. [Approach 1](https://github.com/grommet/grommet/pull/5623). [Approach 2](https://github.com/grommet/grommet/pull/5626).
- Improved DateInput and Calendar keyboard accessibility behavior.
- Provided [additional accessibility resources](https://design-system.hpe.design/foundation/accessibility#internal-resources) from the Product Accessibility Office on the Accessibility page.
- Implemented improved site search capabilities on the HPE Design System site.
- Completed categorization and naming for Navigation patterns.
- Released [Grommet v2.18.0](https://github.com/grommet/grommet/releases/tag/v2.18.0).


```

File: aries-site/src/pages/whats-new.mdx [hunk 2]
```diff
@@ -52,6 +52,10 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## September 20 - October 1, 2021
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:55]:
```suggestion
## September 20 - October 1, 2021
- Kicked off the HPE Design System Council, a steering committee for defining key initiatives and priorities.
- Began holding Design System "office hours" each Wednesday morning. Bring your questions and topics for discussion. Contact the #hpe-design-system channel on Slack to request a calendar invite.
```

Comment (jcfilben) [reply aries-site/src/pages/whats-new.mdx:55]: Good idea to include these!

File: aries-site/src/pages/whats-new.mdx [hunk 3]
```diff
@@ -52,6 +52,10 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+- New resources and a checklist has been added to the [Accessibility page](/foundation/accessibility).
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:58]: Can we link to these resources directly? Lots of text on that page. It'd be nice to have a shortcut.

Comment (jcfilben) [reply aries-site/src/pages/whats-new.mdx:58]: Yeah I will update the link

**Review by halocline — APPROVED** [review]
Approved. @jcfilben , go ahead and "Squash and Merge" once the pre-view build completes successfully.

**Pull request merged**